integration_test/testdata/konfig.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
```

Comment:
I wonder if we can use the index prep  on each test and use this file from there? Or http.Get this from github?
I am not against this but the amount of testdata files might soon get out of hand.

Reply:
In principle, I like the idea to take this from the index prep. However, this particular yaml is an old version which is also needed in the `krew upgrade` test #242 (I made sure it's the same). So I'm reusing this file from there. Besides, having this yaml in the repo makes it more visible which will make test maintenance simpler. So again, I'm still inclined to keep the testdata as-is.

Reply:
OK SG.